lib/schematic/stages/AddLibrarySymbolsStage.ts

```diff
@@ -319,12 +320,17 @@ export class AddLibrarySymbolsStage extends ConverterStage<
   private createPolylineFromPoints(
     points: Array<{ x: number; y: number }>,
     scale: number,
+    center: { x: number; y: number } | undefined,
     fillType: "none" | "background" = "none",
```

Comment:
we have a 2 parameter rule, if there's more than 2 parameters you must switch to named parameters